vyper/semantics/types/indexable/sequence.py

```diff
@@ -97,6 +117,35 @@ def __repr__(self):
     def abi_type(self) -> ABIType:
         return ABI_StaticArray(self.value_type.abi_type, self.length)
 
+    @property
+    def json_abi_typename(self):
+        return f"{self.value_type.json_abi_typename}[{self.length}]"
+
+    def json_abi_type(self, name=""):
```

Comment:
looking better, still not a huge fan of the repeated code between this and dynarray code

Reply:
An alternative is to consolidate the code `json_abi_type` for `ArrayDefinition`, `DynamicArrayDefinition` and `TupleDefinition` under `_SequenceDefinition`. would that be better?